REFERENCE.md

```
@@ -6453,4 +6581,962 @@ This method takes no parameters.
 </dl>
 </details>
 
-<\!-- markdownlint-enable MD024 -->
+## Tokens
+
+<details><summary><code>client.tokens().<a href="https://github.com/scalekit-inc/scalekit-sdk-java/blob/main/src/main/java/com/scalekit/api/TokenClient.java">create</a>(organizationId) -> CreateTokenResponse</code></summary>
+<dl>
+<dd>
+
+#### 📝 Description
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)